### PR TITLE
Update to the latest docker and add the insecure registry option

### PR DESCRIPTION
**Pull request by MiLk**

### CODE REVIEW

File: tasks/rh6.yml [hunk 1]
```diff
@@ -30,5 +30,13 @@
...
+    state=latest
   notify: Restart Docker
+
+- name: "Change docker options"
+  lineinfile: >
+    line='other_args={{ docker_opts }} {{docker_insecure_registry}}'
+    regexp='^other_args='
+    dest=/etc/sysconfig/docker
+    state=present
+  when: docker_opts != '' and docker_inscure_registry != ''
```
Comment (christopherobin) [inline tasks/rh6.yml]: `docker_insecure_registry` instead of `docker_inscure_registry`

File: tasks/rh7.yml [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+  lineinfile: >
+    line='OPTIONS={{ docker_opts }}'
+    regexp='^OPTIONS='
+    dest=/etc/sysconfig/docker
+    state=present
+  when: docker_opts != ''
+
+- name: "Change docker options"
+  lineinfile: >
+    line='INSECURE_REGISTRY={{ docker_insecure_registry }}'
```
Comment (MiLk) [inline tasks/rh7.yml:20]: expect a registry url

File: tasks/rh6.yml [hunk 3]
```diff
@@ -30,5 +30,13 @@
   yum: >
     name=docker-io
     enablerepo=epel
-    state=present
+    state=latest
   notify: Restart Docker
+
+- name: "Change docker options"
+  lineinfile: >
+    line='other_args={{ docker_opts }} {{docker_insecure_registry}}'
```
Comment (MiLk) [inline tasks/rh6.yml:38]: expect the flag + the registry url?

**Comment by MiLk** [comment]
Docker should be installed from the official repository. https://docs.docker.com/engine/installation/centos/

**Pull request closed without merging**